johnsonandjohnson/Bodiless-JS
# fix(test-site): Change gallery-final header to landscape image

## Changes
Fix gallery image to use landscapelinkable for the header image

## Test Instructions

- Static: Verify /gallery-final page uses landscape iamge
- In edit mode, 
  - verify image is editable
  - verify link is editable 
- Verify documentation is updated Site Build Basics (reference both traditional way and use of LandscapeLinkableImage
## Related Issues
